### PR TITLE
Document event publishing via HTTP API.

### PR DESCRIPTION
**Pull request by smn**
It's possible but undocumented.

### CODE REVIEW

**Comment by smn** [comment]
Places where we reference `metrics_store` I renamed it to `metric_store` to be consistent with jsbox.

File: go/apps/http_api_nostream/view_definition.py [hunk 1]
```diff
@@ -14,15 +17,19 @@ class TokenForm(forms.Form):
     push_event_url = forms.CharField(
         help_text='The URL to forward events to via HTTP POST.',
         required=True)
+    metric_store = forms.CharField(
+        help_text='Which store to publish metrics to.',
+        required=False)
```
Comment (justinvdm) [inline go/apps/http_api_nostream/view_definition.py:21]: Can't we put the default here?

Comment (justinvdm) [inline go/apps/http_api_nostream/view_definition.py:21]: Oh actually, that wouldn't work, sorry.

File: go/apps/http_api_nostream/resource.py [hunk 2]
```diff
@@ -260,7 +260,7 @@ def handle_PUT(self, request):
 
         conversation = yield self.get_conversation(user_account)
         store = self.worker.get_api_config(
-            conversation, 'metrics_store', self.DEFAULT_STORE_NAME)
+            conversation, 'metric_store', self.DEFAULT_STORE_NAME)
```
Comment (justinvdm) [inline go/apps/http_api_nostream/resource.py]: Any way we could use the view definitions default, or use the resource's default from the view definition. We save us confusion if the two defaults got out of sync.

**Comment by justinvdm** [comment]
Looks good. One small comment.

File: go/apps/http_api_nostream/resource.py (hunk 2 above)
Comment (smn) [inline go/apps/http_api_nostream/resource.py]: I didn't know there was such a thing.

**Comment by smn** [comment]
@justinvdm ready for re-review

**Comment by justinvdm** [comment]
:+1: Not sure if its worth waiting for Travis, your call.

**Comment by justinvdm** [comment]
Actually, travis hasn't passed yet, maybe its worth waiting?

**Comment by smn** [comment]
travis failing because of unrelated wikipedia failures.

**Comment by justinvdm** [comment]
:+1:

**Pull request merged**